### ISSUE TITLE
Consider offering padding-less versions too

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/fticons#99]
## What
IIRC the large areas of padding on the icons was to accommodate icons which may be of quite different shapes. An unfortunate downside of this is that it makes them harder to use for some common use cases e.g. put the icon in a coloured container with just a little bit of padding around it. It's be good to make no/less padding versions available particularly as most (all?) of the icon set no longer have strange dimensions. Maybe even a precursor to only providing less padded versions in a future major.

e.g these circles are just the right size, but the icons are too small. Scaling the icons up without bigger circles requires extra markup + some CSS hacks. If the icon was less padded it's be simple
![image](https://user-images.githubusercontent.com/447559/85849221-7a243680-b7a2-11ea-9e34-d14eb112116d.png)

**Comment by notlee** [comment]
Yeah I agree we should do this and update the [o-icons](https://github.com/Financial-Times/o-icons) api accordingly, with a view of making it the default and deprecating the padded versions. 

We already use CSS negative margins in a few components to negate the inbuilt white space and it's very frustrating ([e.g. o-share](https://github.com/Financial-Times/o-share/blob/efeaedab9ca0f801dcc47682f0fdabe01d1e48dc/src/scss/_mixins.scss#L142)). 

Does anyone know why the [guidelines were added](https://github.com/Financial-Times/fticons/pull/8/commits/ffd4c3c488bc0bc315f31f50a82721a41bde516c)? Maybe to force a minimum whitepsace around icons? If so we can encourage a consistent whitespace with CSS in [o-icons](https://github.com/Financial-Times/o-icons).

**Comment by notlee** [comment]
Mark Limb says

>Having a small margin allows for irregular shapes to exist within the same overall space
>But I agree that the margin is pretty large and not always helpful when aligning elements
>Scope creep.... but it would be great to revisit the icons at some point just to soften them up and refine a bit

![image](https://user-images.githubusercontent.com/10405691/86118996-99380680-bac9-11ea-97fd-b3293bcb592c.png)

And points to Material 
https://material.io/design/iconography/system-icons.html#grid-and-keyline-shapes

So this may be a case of reducing rather than removing the whitespace around icons

**Comment by notlee** [comment]
This continues to be an issue for Origami users:
https://financialtimes.slack.com/archives/C02FU5ARJ/p1619014951106000

Here are a couple of examples of Origami components hacking around the problem:
https://github.com/Financial-Times/o-message/blob/fe07135fbffabeb885a63c0cc2e6f64b94abe5cb/main.scss#L152
https://github.com/Financial-Times/o-typography/blob/57f2c8b395c6e71c8d142cdba893e491925d1b76/src/scss/use-cases/_general.scss#L164
